scripts/update-x.x.x-x.x.x.sql

```diff
@@ -0,0 +1,61 @@
+-- This script contains multiple comment blocks that can update
+-- APEL version x.x.x databases of the following types to x.x.x:
+--  - Cloud Accounting Database
+
+/*
+-- UPDATE SCRIPT FOR CLOUD SUMMARY SCHEMA
+
+-- If you have a Cloud Accounting Database and wish to
+-- upgrade to APEL Version next, remove the block comment
+-- symbols around this section and run this script
+
+-- This section will:
+-- - Remove CloudType from the CloudSummaries primary key. This
+--   ensures new summaries overwrite old summaries after updates
+--   to CASO version name.
+
+DROP TABLE IF EXISTS CloudSummaries;
+CREATE TABLE CloudSummaries (
```

Comment:
We can't drop and recreate an existing table as part of an update schema as that'll drop the data it contains! This should be one or more `ALTER TABLE` statements to change the primary key.